testpar/t_filters_parallel.c

```diff
@@ -576,7 +576,7 @@ create_datasets(hid_t parent_obj_id, const char *dset_name, hid_t type_id, hid_t
             dset_name_ptr = dset_name_multi_buf;
             n_dsets       = (rand() % (MAX_NUM_DSETS_MULTI - 1)) + 2;
 
-            /* Select between 1 and (n_dsets - 1) datasets to NOT be filtered */
+            /* Select between 1 and (n_dsets - 1) datasets to unfiltered */
```

Comment:
`to unfiltered` -> `to be unfiltered`, but there's nothing wrong with the sentence as is now

Reply:
Agreed, there is nothing grammatically incorrect with the current statement.